clang/lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -5935,3 +5918,28 @@ llvm::DINode::DIFlags CGDebugInfo::getCallSiteRelatedAttrs() const {
 
   return llvm::DINode::FlagAllCallsDescribed;
 }
+
+llvm::DIExpression *
+CGDebugInfo::createConstantValueExpression(clang::ValueDecl const *VD,
+                                           const APValue &Val) {
+  llvm::DIExpression *ValExpr = nullptr;
```

Comment:
I know this is how the original code was written, but since we are making a separate function we are now in a position where we can delete this line, early-return on the `if` and replace all assignments to `ValExpr` with return statements.

Reply:
will do